database/Reseed/reseed.js

```diff
@@ -1,18 +1,29 @@
 const Chance = require('chance');
 const fs = require('fs');
-
+const { Readable, pipeline } = require('stream');
+let startTime = Date.now();
+let nearFile = fs.createWriteStream('nearby.txt', {flags: 'a'})
+    .on('error', (err) => console.error(err.message));
+let experienceFile = fs.createWriteStream('experience.txt', {flags: 'a'})
+    .on('error', (err) => console.error(err.message));
+let restFile = fs.createWriteStream('rest.txt', {flags: 'a'})
+    .on('error', (err) => console.error(err.message));
+let attractionFile = fs.createWriteStream('attraction.txt', {flags: 'a'})
+    .on('error', (err) => console.error(err.message));
 
 const chance = new Chance();
 
 const seedData = [];
 
 const attractionIds = [];
 
+let locations = [];
+
 const d = '|'; //delimiter
 
 const qty = 10000000; //quantity of records
 
-for (let i = 9000001; i <= qty; i += 1) {
+for (let i = 1; i <= qty; i += 1) {
```

Comment:
I am not sure if this has much of an impact, but creating an array of 10M records for the padded number requires you to iterate over the list of 10M twice (here and then in the pipe process) - could factor the building the padded id into the pipe stream and in line 106 forloop just declare a limit of 10M entries.